app/models/abuse_report.rb

```diff
@@ -49,4 +65,23 @@ def send_report
     )
     reporter.send_report!
   end
+
+  # if the URL clearly belongs to a work (i.e. contains "works/123")
+  # make sure it isn't reported more than ABUSE_REPORTS_PER_WORK_MAX times per month
+  def work_is_not_over_reported
+    if url.match(/works\/\d+/)
+      # use "works/123/" instead of just the id to avoid confusion with chapter ids
+      work_params_only = url.match(/works\/\d+\//).to_s
+      existing_reports_total = AbuseReport.where("created_at > ? AND
+                                                 url LIKE ?",
+                                                 1.month.ago,
+                                                 "%#{work_params_only}%").
```

Comment:
I didn't think it was worth going back and cleaning up all the URLs, since I suspect there are a lot of them... but I realized I could do it for just the past month, which are the ones that matter, so I've added a rake task to do that.

I don't think I can add `^` to this -- the "works/123/" isn't going to be at the beginning of the string, but rather somewhere in the middle of it. However, I did change it so it will be looking for "/works/123/" instead. That way, "external_works/123/" won't be a match.